ush/templates/FV3.input.yml

```diff
@@ -52,18 +52,24 @@ FV3_HRRR:
     kord_tm: -9
     kord_tr: 9
     kord_wz: 9
+    nord_tr: 2
     nrows_blend: 10
     regional_bcs_from_gsi: false
     write_restart_with_bcs: false
   gfs_physics_nml:
     <<: *gsd_sar_phys
     cdmbgwd: [3.5, 1.0]
     do_mynnsfclay: true
+    do_sfcperts: !!python/none
     effr_in: true
     gwd_opt: 3
     imfdeepcnv: -1
     imfshalcnv: -1
     lsoil: 9
+    iaer: 5111
```

Comment:
The old value was 111, will 5111 replace it?

Reply:
Yes, it will be 5111 in the resulting namelist.